CONTRIBUTING.md

```diff
@@ -1,4 +1,41 @@
-## Releasing a new version
+# Development
+
+Set up an alias for the EAS CLI so you can try it in projects all around your computer. The project is compiled on the fly so you don't need to run a build (watch) command. Open your `.zshrc` or other config file and add:
+
+```
+alias easd="/PATH/TO/eas-cli/bin/run"
+```
+
+Then use it with `easd` like `easd build`.
+
+Optional: to start the build command in watch mode run:
+
+```
+yarn start
+```
+
```

Comment:
Maybe link to https://github.com/expo/eas-build/blob/main/DEVELOPMENT.md on how to setup local builds for development